src/clients/MySQLClient.ts

```diff
@@ -0,0 +1,29 @@
+import { DBClient } from './DBClient';
+import mysql2, { Connection } from 'mysql2/promise';
+
+export class MySQLClient implements DBClient {
+
+    public connection: Connection | null = null;
+    private readonly config: any;
+    private readonly connectError = new Error('Connection is not established! Call connect method!');
```

Comment:
I guess it is a constant so maybe it should be named as a constant in that case?

Reply:
make sense

Reply:
fixed